### DISCUSSION TITLE
Welcome to python-garminconnect Discussions!

### DISCUSSION THREAD

**Discussion by cyberjunky** [cyberjunky/python-garminconnect#181] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jisaac01** [comment]
Hey, I want to thank you for putting this together. 
I did a massive project I wouldn't have been able to do without this tool. 
I deal with chronic pain, and wanted to try to compare my sleep data over time (as a proxy for pain levels) against various medications and foods that I track. 
I was able to reload old Garmin data for a couple of years back and get the detailed data I needed. 
The fact that Garmin only allows you to reload 30 days at a time would have made this even more impossible if I hadn't been able to script it, and in addition the archive they give you doesn't have the sleep breakdown I wanted. 
Thanks!

**Comment by C-Pisani** [comment]
hi @cyberjunky,

Thanks for putting this together, it is extremely useful. I would like to ask you if it is possible to pass a hashed password to the Garmin login() function. I want to retrieve a hashed password but cannot login. FYI, I'm a student, so my knowledge is a bit restricted in this area.

Thanks

**Comment by hky284** [comment]
@cyberjunky Are the Vo2 Max and/or fitness age metric accessible? I am not able to find it. 

P.S: Thank you for the amazing API for Garmin Connect. It has been a great help for a school project.
